kover-gradle-plugin/docs/index.md

```diff
@@ -36,11 +42,638 @@ Gradle plugin that measures the coverage for tests running on the JVM and genera
 * Using JaCoCo library as an alternative for coverage measuring and report generation.
 
 ## Quickstart
+To make the minimal configuration of the plugin, determine the type of your Gradle build and learn the instructions for configuring
+
+## Build types
+### Single module Kotlin JVM project
+Please refer to the [example project](https://github.com/Kotlin/kotlinx-kover/tree/main/kover-gradle-plugin/examples/jvm/single).
+
+#### Application of Kover plugin
+[Apply the plugin](#apply-kover-gradle-plugin-in-project) in a single project in the Gradle build.
```

Comment:
I would insert recommended way of applying plugin right here, so user won't need to jump back and forth between document's sections.